metarParser-services/src/main/java/io/github/mivek/service/MetarService.java

```diff
@@ -45,6 +45,9 @@ public Metar retrieveFromAirport(final String icao) throws ParseException, IOExc
         HttpResponse<Stream<String>> response = HttpClient.newBuilder()
                 .build()
                 .send(request, HttpResponse.BodyHandlers.ofLines());
+
```

Comment:
I am not a fan of the duplicated code here but you are not responsible for this.